compile warning

after i run compile command for main it throws this warning,

```
func.c: In function ‘editClass’:
func.c:781:17: warning: implicit declaration of function ‘displayclasslist’ [-Wimplicit-function-declaration]
  781 |                 displayclasslist();
      |                 ^~~~~~~~~~~~~~~~
func.c: At top level:
func.c:821:6: warning: conflicting types for ‘displayclasslist’
  821 | void displayclasslist(){
      |      ^~~~~~~~~~~~~~~~
func.c:781:17: note: previous implicit declaration of ‘displayclasslist’ was here
  781 |                 displayclasslist();
```

--- a/func.c
+++ b/func.c
@@ -763,7 +763,7 @@ void editClass(){
 
         //  printf("\n===Edit Class===\n\n");
          printf("\t1. Edit Class Details\n");
-         printf("\t2. Delete Class\n");
+         printf("\t2. Display Class\n");
          printf("\t3. Back to Manage Class Menu\n");
          printf("\t0. Back to Main Menu\n");
          printf("======================================\n");
@@ -778,9 +778,7 @@ void editClass(){
             break;
 
             case 2:
-                printf("[-] Enter Class ID to Delete: ");
-                scanf("%d",&classID);
-                deleteClass(classID);
+                displayclasslist();
                 break;
             case 3:
                  manageClass();

--- a/func.c
+++ b/func.c
@@ -353,8 +353,15 @@ void stdReg()
     char contnumber[15];
     char email[50];
 
-    printf("[-] Enter Student ID: ");
-    scanf("%d", &stid);
+    const char *query = "SELECT MAX(StudentID) FROM Students;";;
+    char **result = fetchData(query, TEXT);
+    if (result != NULL) {
+        stid = atoi(result[0]);
+        free(result[0]);
+        free(result);
+    }
+
+    printf("[|] Enter Data for Student ID: %d\n", stid + 1);
 
     printf("[-] Enter First Name: ");
     scanf("%s", firstname);
@@ -372,7 +379,7 @@ void stdReg()
     scanf("%s", email);
 
     char *data;
-    asprintf(&data, "%d, '%s', '%s', '%s', '%s', '%s'", stid, firstname, lastname, dob, contnumber, email);
+    asprintf(&data, "%d, '%s', '%s', '%s', '%s', '%s'", stid + 1, firstname, lastname, dob, contnumber, email);
 
     // Write user details to a text file
     FILE *file = fopen("user_log.txt", "a"); // Open file in append mode
@@ -405,9 +412,15 @@ void tutReg()
     char contnumber[15];
     char email[50];
 
-    printf("[-] Enter Tutor ID: ");
-    scanf("%d", &tutid);
-    getchar();
+    const char *query = "SELECT MAX(TutorID) FROM Tutors;";;
+    char **result = fetchData(query, TEXT);
+    if (result != NULL) {
+        tutid = atoi(result[0]);
+        free(result[0]);
+        free(result);
+    }
+
+    printf("[|] Enter Data for Tutor ID: %d\n", tutid + 1);
 
     printf("[-] Enter First Name: ");
     fgets(firstname, sizeof(firstname), stdin);
@@ -430,7 +443,7 @@ void tutReg()
     email[strcspn(email, "\n")] = '\0';
 
     char *data;
-    asprintf(&data, "%d, '%s', '%s', '%s', '%s', '%s'", tutid, firstname, lastname, subject, contnumber, email);
+    asprintf(&data, "%d, '%s', '%s', '%s', '%s', '%s'", tutid + 1, firstname, lastname, subject, contnumber, email);
 
 
     // You should ask to verify the data before inserting

--- a/func.c
+++ b/func.c
@@ -1224,6 +1224,7 @@ void insertEnrollmentData(int stid)
     sprintf(insertSQL, "INSERT INTO Enrollments (EnrollmentID, StudentID, ClassID, EnrollmentDate) VALUES (%d, %d, %d, '%s');", lastEnrollmentID + 1, stid, classID, enrollmentDay);
     if (executeSQL(db, insertSQL) == SQLITE_OK)
     {
+        printf("=======================================\n");
         printf("[+] Enrollment Data Inserted Successfully.\n");
     }
     else
@@ -1279,6 +1280,7 @@ void insertClassData(int tutorid)
     sprintf(insertSQL, "INSERT INTO Classes (ClassID, ClassName, TutorID, ClassTime, ClassDays) VALUES (%d, '%s', %d, '%s', '%s');", lastClassID + 1, className, tutorid, classTime, classDays);
     if (executeSQL(db, insertSQL) == SQLITE_OK)
     {
+        printf("=======================================\n");
         printf("[+] Class Data Inserted Successfully.\n");
     }
     else

--- a/func.c
+++ b/func.c
@@ -932,6 +932,7 @@ void manageTutors() {
 
 void manageClass() {
     int classChoice;
+    int classID;
     do {
         printf("============ Manage  Class ============\n");
         printf("1. Add Class\n");
@@ -952,7 +953,9 @@ void manageClass() {
                 break;
             case 3:
                 // Add related function
-                displayclasslist();
+                printf("[-] Enter Class ID to Delete: ");
+                scanf("%d",&classID);
+                deleteClass(classID);
                 break;
             case 0:
                 printf("[|] Returning to main menu.\n");

--- a/func.h
+++ b/func.h
@@ -43,5 +43,6 @@ void checkAttendance();
 void status();
 void insertEnrollmentData(int stid);
 void insertClassData(int tutorid);
+void displayclasslist();
 
 #endif

